cypress/fixtures/fireboltCalls/accessibility.json

```diff
@@ -481,6 +481,11 @@
         "validationJsonPath": "eventResponse.styles.fontFamily",
         "content": "ACCESSIBILITY_FONTFAMILY_MONOSPACE"
     },
+    "ONCLOSEDCAPTIONSSETTINGSCHANGED_WITH_DEFAULT_VALUE_FOR_FONTFAMILY": {
+        "event": "accessibility.onClosedCaptionsSettingsChanged",
+        "validationJsonPath": "eventResponse.styles.fontFamily",
+        "content": "ACCESSIBILITY_FONTFAMILY_VALUE_FOR_NULL_PARAMS"
+    },
```

Comment:
Initially when new json objects for undefined was added, I accidently removed the corresponding existing objects that validates against a specific value instead of undefined. Later realised that existing objects are still used by other scenarios, so added those back. May be thats why the file looks confusing in github ui
To summarise, as part of this ticket only the new json objects required for undefined validation has been added for both event and method, existing ones are untouched as they are still used by other scenarios
eg: existing object used in positive scenario -
"ONCLOSEDCAPTIONSSETTINGSCHANGED_WITH_40_FOR_WINDOWOPACITY": {
        "event": "accessibility.onClosedCaptionsSettingsChanged",
        "validationJsonPath": "eventResponse.styles.windowOpacity",
        "content": "INTEGER40"
    },
    new json object used for null scenario -
     "ONCLOSEDCAPTIONSSETTINGSCHANGED_WITH_DEFAULT_VALUE_FOR_WINDOWOPACITY": {
        "event": "accessibility.onClosedCaptionsSettingsChanged",
        "validationJsonPath": "eventResponse.styles.windowOpacity",
        "content": "ACCESSIBILITY_WINDOWOPACITY_VALUE_FOR_NULL_PARAMS"
    },

Reply:
Okay. if existing ones are used, then we cannot remove them